testsuite/performance/infinispan/pom.xml

```diff
@@ -0,0 +1,213 @@
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <groupId>org.keycloak.testsuite</groupId>
+        <artifactId>performance</artifactId>
+        <version>4.0.0.CR1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>performance-keycloak-infinispan-server</artifactId>
+    <name>Keycloak Performance TestSuite - Infinispan Server</name>
+    <packaging>pom</packaging>
+    
+    <properties>
+        <infinispan.groupId>org.infinispan.server</infinispan.groupId>
+        <infinispan.artifactId>infinispan-server-build</infinispan.artifactId>
+        <infinispan.version>8.2.8.Final</infinispan.version>
+        <infinispan.unpacked.folder.name>infinispan-server-${infinispan.version}</infinispan.unpacked.folder.name>
+        <infinispan.unpacked.home.dc1>${project.build.directory}/dc1/${infinispan.unpacked.folder.name}</infinispan.unpacked.home.dc1>
+        <infinispan.unpacked.home.dc2>${project.build.directory}/dc2/${infinispan.unpacked.folder.name}</infinispan.unpacked.home.dc2>
+        
+        <script.extension>sh</script.extension>
+        <jboss.cli.script>ispn-cli.${script.extension}</jboss.cli.script>
+        <add.user.script>add-user.${script.extension}</add.user.script>
+        
+        <scripts.dir>${project.build.scriptSourceDirectory}</scripts.dir>
+        <resources.dir>${project.basedir}/src/main/resources</resources.dir>
+        
+        <skip.add.management.user>true</skip.add.management.user>
+        <skip.docker.config>false</skip.docker.config>
+    </properties>
+
+    <build>
+        
+        <plugins>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack-infinispan-servers</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <overWriteIfNewer>true</overWriteIfNewer>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>${infinispan.groupId}</groupId>
+                                    <artifactId>${infinispan.artifactId}</artifactId>
+                                    <version>${infinispan.version}</version>
+                                    <type>zip</type>
+                                    <outputDirectory>${project.build.directory}/dc1</outputDirectory>
```

Comment:
Currently this extracts 2 infinispan servers into two different locations (dc1, dc2) and then updates clustered.xml just in the same way. Can you update it to extract it just once instead and then work on configuration clustered-dc*X*.xml instead, and run the appropriate configuration file in docker?